textproc/xdxf_makedict/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+#
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        soshial xdxf_makedict a26b847fa785ccb87a0b67940c824e29da97c2fd
+version             20160728
```

Comment:
The progrem identifies itself as 0.4.1-beta1, so it makes more sense to use that one before the additional date. In the meantime another commit came in.

I'll fix this and merge the commit.